.envrc

```diff
@@ -25,11 +25,12 @@ export FISSILE_RELEASE="${FISSILE_RELEASE#,}"
 export FISSILE_ROLE_MANIFEST="${PWD}/role-manifest.yml"
 export FISSILE_LIGHT_OPINIONS="${PWD}/opinions.yml"
 export FISSILE_DARK_OPINIONS="${PWD}/dark-opinions.yml"
-export FISSILE_WORK_DIR="${PWD}/work-dir"
+export FISSILE_WORK_DIR="$HOME/.fissile/uaa-work-dir"
 export FISSILE_REPOSITORY="uaa"
+export FISSILE_CACHE_DIR="$HOME/.bosh/cache"
```

Comment:
I think it got the value from `scf`, because uaa is nested within in my setup (like it will be as a submodule)